wgengine/magicsock/magicsock.go

```diff
@@ -1295,7 +1296,7 @@ func (c *Conn) receiveIP(b []byte, ipp netip.AddrPort, cache *ippEndpointCache)
 		de, ok := c.peerMap.endpointForIPPort(ipp)
 		c.mu.Unlock()
 		if !ok {
-			return nil, false
+			return &lazyEndpoint{c: c, src: ipp}, true
```

Comment:
Should we have a nodeAttr to turn it off?

Reply:
Normally I would say yes, but this is only used if we were already going to fail, so what's the worst that can happen?

Unless this panics and takes down the process.

Reply:
Eh it's cheap enough to add in case we do see something completely unexpected.